docs/source/tutorials/index.rst

```diff
@@ -15,4 +15,5 @@ Tutorials
    multilabel_classification
    token_classification
    pred_probs_cross_val
+   datalab/
```

Comment:
Consistently order where datalab tutorials appear in the side bars.
My personal pref is for them to appear higher up in the side bar.